### PR TITLE
Crated script for cases if PreloadedPT have application with NULL value of policies

### PR DESCRIPTION
**Pull request by Aleksandriys**
@istoimenova and @maria-mih please review.
This script cover these req. APPLINK-16253 and APPLINK-19829.
 Iliana second_test have status PartlyReady I need your help with sent all avaliable RPCs, I sended 22 RPCs and received DISALLOWED responce, it is OK for the test. 
But I have 13 RPCs which come with INVALID_DATA (they commented) and 3 RPCs come with INVALID_DATA also but with  error "mandatory parameter spaceAvailable not present" - they also commented for run script.
Thank you for the help.

### CODE REVIEW

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 1]
```diff
@@ -0,0 +1,111 @@
+----- [GeneralResultCode] DISALLOWED. A request comes with appID which has "null" permissions in Policy Table
+----- In case PolicyTable has "<appID>": "null" in the Local PolicyTable for the specified application with appID,
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: this should be in section --Description

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 2]
```diff
@@ -0,0 +1,111 @@
+----- [GeneralResultCode] DISALLOWED. A request comes with appID which has "null" permissions in Policy Table
+----- In case PolicyTable has "<appID>": "null" in the Local PolicyTable for the specified application with appID, 
+----- PoliciesManager must return DISALLOWED resultCode and success:"false" to any RPC requested by such <appID> app.
+
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: Please also add 1. Used preconditions (if applicable)
2. Performed steps 
3. Expected result 

Please check https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/27 for more details

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+-- {rpc = "ListFiles", params = {}},
+ --{rpc = "PutFile", params = {syncFileName ="icon.png", fileType ="GRAPHIC_PNG", persistentFile =false, systemFile = false, offset =0, length =11600}},
+ --{rpc = "DeleteFile", params = {syncFileName = "test.png"}},
+}
+
+--[[ General Precondition before ATF start ]]
+--Copy attached json file in SDL_bin folder.
+--Json contains application 123abc with NULL policies
+commonSteps:DeleteLogsFileAndPolicyTable()
+--[[ Preconditions ]]
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: Please add below line:
commonFunctions:newTestCasesGroup("Preconditions")

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Test = require('connecttest')
+function Test:OpenNewSession()
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: Please add to all functions in this section prefix Preconditon_ e.g.

function Test:Preconditon_ OpenNewSession

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+commonFunctions:userPrint(33, "================= Precondition ==================")
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: Please remove the print should be called as indicated above. It is put immediately after name of section.

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+  self.mobileSession2 = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession2:StartService(7)
+end
+--[[Test_1]]
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: This should be in section --[[ Test ]] followed by 
commonFunctions:newTestCasesGroup("Test")

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+function Test:RAI_InNewSession_WithNULL_Polices()
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: For tests in Test section please add prefix TestStep_
e.g.  Test:TestStep_RAI_InNewSession_WithNULL_Polices()

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+commonFunctions:userPrint(33, "================= Test_Case1 ====================")
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: This line is redundant, name of function itself will indicate it is test step.

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+      carrier = "Megafon",
+      firmwareRev = "Name: Linux, Version: 3.4.0-perf",
+      osVersion = "4.4.2",
+    }
+  }
+local corId = self.mobileSession2:SendRPC("RegisterAppInterface", registerAppInterfaceParams)
+  self.mobileSession2:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+end
+
+--[[Test_2]]
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: Please see comments above.

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua (hunk 6 above)
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: Remove line

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+function Test:CheckRPCs_ForApp_withPolicyNull()
+  commonFunctions:userPrint(33, "================= Test_Case2 ====================")
+ for i = 1 , #RPC do
+  local corId = self.mobileSession2:SendRPC(RPC[i].rpc, RPC[i].params) 
+  self.mobileSession2:ExpectResponse(corId, {success = false, resultCode = "DISALLOWED" })
+end
+end
+--[[ Postconditions ]]
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: Please organize section like shown in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/27/files

File: test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+      osVersion = "4.4.2",
+    }
+  }
+local corId = self.mobileSession2:SendRPC("RegisterAppInterface", registerAppInterfaceParams)
+  self.mobileSession2:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+end
+
+--[[Test_2]]
+function Test:CheckRPCs_ForApp_withPolicyNull()
+  commonFunctions:userPrint(33, "================= Test_Case2 ====================")
```
Comment (maria-mih) [inline test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua]: See comment above

**Comment by Aleksandriys** [comment]
@maria-mih please review

**Pull request closed without merging**